library/VTKExtensions/Rendering/vtkF3DInteractorStyle.cxx

```diff
@@ -114,7 +114,24 @@ void vtkF3DInteractorStyle::Dolly()
   {
     return;
   }
-  this->Superclass::Dolly();
+
+  assert(this->CurrentRenderer != nullptr);
+
+  vtkRenderWindowInteractor* rwi = this->Interactor;
+  const double* center = this->CurrentRenderer->GetCenter();
+  const int* current_position = rwi->GetEventPosition();
+  const int* last_position = rwi->GetLastEventPosition();
+  const int dy = current_position[1] - last_position[1];
+  const int dx = current_position[0] - last_position[0];
+  const double dxf = this->MotionFactor * dx / center[0];
+  const double dyf = this->MotionFactor * dy / center[1];
+  auto dtf = std::abs(dyf) > std::abs(dxf) ? dyf : dxf;
```

Comment:
```suggestion
  double dtf = std::abs(dyf) > std::abs(dxf) ? dyf : dxf;
```

Reply:
tbh the `const double` and `const int` are not needed imo but I wont push for it. We just had a lengthy discussion about it in discord, I will ping you if you are interested @DeveloperPaul123